x-pack/plugin/profiler/src/main/java/org/elasticsearch/xpack/profiler/StackFrame.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.profiler;
+
+import org.elasticsearch.xcontent.ToXContent;
+import org.elasticsearch.xcontent.XContentBuilder;
+import org.elasticsearch.xpack.profiler.utils.MapExtractor;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Objects;
+
+final class StackFrame implements ToXContent {
```

Comment:
this always has a top level anonymous object, then it should be a ToXContentObject.

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/91640/commits/c21262b78a3faa152e865a7e834ff858c4cc3f10.